token-group/js/package.json

```diff
@@ -47,11 +47,8 @@
         "@solana/web3.js": "^1.91.0"
     },
     "dependencies": {
-        "@solana/codecs-core": "2.0.0-preview.1.20240308041540.58947c719d61c07ce3b1ac04edf51a2b8fa4a3eb",
-        "@solana/codecs-data-structures": "2.0.0-preview.1.20240308041540.58947c719d61c07ce3b1ac04edf51a2b8fa4a3eb",
-        "@solana/codecs-numbers": "2.0.0-preview.1.20240308041540.58947c719d61c07ce3b1ac04edf51a2b8fa4a3eb",
-        "@solana/codecs-strings": "2.0.0-preview.1.20240308041540.58947c719d61c07ce3b1ac04edf51a2b8fa4a3eb",
-        "@solana/options": "2.0.0-preview.1.20240308041540.58947c719d61c07ce3b1ac04edf51a2b8fa4a3eb",
+        "@solana/codecs": "2.0.0-preview.1",
+        "@solana/options": "2.0.0-preview.1",
```

Comment:
I know it's not super intuitive but `@solana/codecs` also contains `@solana-options`. We should probably rename the latter `@solana/codecs-options` because 70% of the package is codecs anyway.

Reply:
Ah ok, will do in follow-up work!